.github/actions/collect-logs/action.yml

```diff
@@ -40,13 +40,17 @@ runs:
         if test -f ~/.kube/config; then
           make tools/bin/kubectl
           mkdir /tmp/test-logs/cluster
+          tools/bin/kubectl get hosts --all-namespaces -o yaml >/tmp/test-logs/cluster/all-hosts.yaml || true
           tools/bin/kubectl get pods --all-namespaces      >/tmp/test-logs/cluster/all-pods.txt || true
           tools/bin/kubectl describe pods --all-namespaces >/tmp/test-logs/cluster/all-pods-described.txt || true
 
           tools/bin/kubectl get pods --all-namespaces -ocustom-columns="name:.metadata.name,namespace:.metadata.namespace" --no-headers | while read -r name namespace; do
             tools/bin/kubectl --namespace="$namespace" logs "$name" >"/tmp/test-logs/cluster/pod.${namespace}.${name}.log" || true
           done
+
+          tools/bin/kubectl cp xfpredirect:/tmp/ambassador/snapshots /tmp/test-logs/cluster/xfpredirect.snapshots || true
         fi
+        cp /tmp/*.yaml /tmp/test-logs || true
```

Comment:
The `|| true` is to force CI not to turn red when unable to copy files, correct?

Reply:
Correct. the reason I expect it to maybe not be able to copy things is if the files don't exist.